app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -282,7 +290,9 @@ public boolean onCreateOptionsMenu(Menu menu) {
             // TODO: used vectors are not compatible with API 19 and below, change them
             menu.findItem(R.id.notifications).setIcon(ContextCompat.getDrawable(this, R.drawable.ic_notification_white_clip_art));
         } else {
-            menu.findItem(R.id.notifications).setIcon(ContextCompat.getDrawable(this, R.drawable.ic_notification_white_clip_art_dot));
+            final View notification=menu.findItem(R.id.notifications).getActionView();
+            txtViewCount=notification.findViewById(R.id.notification_count_badge);
```

Comment:
Use butterknife. 

https://github.com/commons-app/apps-android-commons/wiki/Code-style#butterknife

Reply:
Does that means declaring it globally like this
```
@BindView(R.id.notifications)
    MenuItem menuItem;

```

Reply:
Sorry, I didn't notice that `txtViewCount` is part of `notification` view. Ignore my comment.

Reply:
OK I'll revert the change